docs/HowToGuides/GettingStarted.md

```diff
@@ -258,21 +262,10 @@ Phew, that's a lot to digest! Now let's proceed to the actual build itself!
      ```
    - Linux:
      ```sh
-     utils/build-script --release-debuginfo --skip-early-swift-driver \
-       --skip-early-swiftsyntax
+     utils/build-script --release-debuginfo
      ```
      If you installed and want to use Sccache, include the `--sccache` option in
      the invocation as well.
-   <!-- FIXME: Without this "hard" line break, the note doesn’t get properly spaced from the bullet -->
-   <br />
-
-   > **Note**  
-   > If you are planning to work on the compiler, but not the parts that are
-   > written in Swift, pass `--bootstrapping=hosttools` to speed up local
-   > development. Note that on Linux — unlike macOS, where the toolchain already
-   > comes with Xcode — this option additionally requires
-   > [a recent Swift toolchain](https://www.swift.org/download/) to be
-   > installed.
```

Comment:
Did we change the default to host tools on macOS too?

Reply:
Good catch, I misread and thought this note was only for linux, will modify it.